### ISSUE TITLE
strikethrough broken

### ISSUE THREAD

**Issue by bsaelim** [markedjs/marked#286]
~~some text~~ is no longer working.

**Comment by r0the** [comment]
I had the same issue, but only when using marked as described under "Pro level", so I investigated a bit:

This works as expected:
```
console.log(marked('~~strikethrough~~'));
```
and renders as:
```
<p><del>strikethrough</del></p>
```

When using the lexer and parser separately with a custom renderer, the following code 

```
var tokens = marked.lexer('~~strikethrough~~', {renderer: myRenderer});
console.log(marked.parser(tokens, {renderer: myRenderer}));
```
ignores the strikethrough and produces:

```
<p>~~strikethrough~~</p>
```
As far as I can see, the `marked.parser` method seems to ignore the `gfm: true` default. The following code:

```
var tokens = marked.lexer('~~strikethrough~~', {renderer: myRenderer});
console.log(marked.parser(tokens, {renderer: myRenderer, gfm: true}));
```
outputs `<p><del>strikethrough</del></p>` again as expected.

So my issue here is that `marked.parser()` doesn't respect the default options.

I have no idea if that is the same issue of @bsaelim, but it may be an explanation.

**Comment by r0the** [comment]
I think the issue as I described it may be resolved by pull request #1015.

**Comment by Feder1co5oave** [comment]
I confirm #1015 solves this issue, but it has a performance problem if you call marked several times. For now, remember to do something like this:
```javascript
var marked = require('marked');
var myRenderer = new marked.Renderer();
marked.setOptions({ renderer: myRenderer });
var tokens = var tokens = marked.lexer('~~strikethrough~~');
console.log(marked.parser(tokens));
```
outputs
```html
<p><del>strikethrough</del></p>
```

I'm closing this in favor of the linked PR.
